script/Ciel/Day 13/13_01_02_CIEL13_7E.txt

```diff
@@ -276,187 +329,222 @@ The boy was speaking the truth.
+Nor is she the unbridled and impetuous Arcueid she'd been after being murdered.
 }
 [sha:b73ee8339115d12c82c47c12de8e90c56a77a885]{
 -- Page 389, Offset 36409.
 -- ……何か未知の<在り方|アバター>。
-...Before his eyes is an avatar that has yet to be seen before.
+This is an entirely new side to the avatar in white...
 }
 [sha:386ebdb827711321267ed2074b4ad83f6332683a]{
 -- Page 389, Offset 36410.
 -- 涼やかに人を<試|ため>し破滅させる、無慈悲な女神のような微笑みだった。
-Her smile is like that of a ruthless goddess, casually testing her people and leading them to their demise.
+Her smile is that of a ruthless goddess;
+ the sort of femme fatal that would casually try and tempt her subjects to their destruction.
 }
 [sha:da8cd878c9b5907a7d61c556abd64866ea3ff33e]{
 -- Page 390, Offset 36411.
 -- 「おい」
+// Oi.
 "Oi."
 }
 [sha:709088eccc22e2c59ea8c92860f6b9853e48eab1]{
 -- Page 390, Offset 36412.
 -- 「でも、その前にこっちが先ね。
-"But first, I have to deal with this.
+"Besides, I'd better get this out of the way first.
 }
 [sha:f7e9fb163c2f4d0de50ce1fb42debadcd95ddf9c]{
 -- Page 390, Offset 36413.
 -- アナタがいるとロアを生かそうとするし、なによりシエルの味方をされたら面倒だし。
-If I let you go, you'll try to keep Roa alive, and more importantly, it'd be troublesome if you ended up teaming up with Ciel.
+If I left you to your own devices you'd try and keep Roa alive,
+ and I can't have you teaming up with Ciel.
 }
 [sha:851096517746ba5ed9e35fc984b495ef158b5d48]{
 -- Page 390, Offset 36414.
 -- 小さい方の邪魔者は、ここで蹴散らしておきましょうか」
-Perhaps I should get rid of the smaller obstacles first."
+// Callback 邪魔者は大小様々...大きい方の邪魔者ね on offset 36375.
+Maybe I'd better dispose of the smaller obstacles first?"
 }
 [sha:1fc54cc5f5ca85cf0eaec936aa7d4d13da6a03fa]{
 -- Page 391, Offset 36415.
 -- ―――月下に不可視の花が咲く。
+// Debating whether this should be plural, if each invisible flower = 1 downed executor
 ―――An invisible flower blooms beneath the moonlight.
 }
 [sha:b526eac7661ce432774cf26386e0429dd8992b72]{
 -- Page 391, Offset 36416.
 -- <少年|マーリオゥ>の合図に応じて、周囲に展開した代行者たちが状況を開始する。
-The Executors surrounding them begin to attack in response to Mario's signal.
+At Mario's signal, the surrounding Executors commence their attack.
 }
 [sha:3f7bcd40cb29529e496a9031bf21523ecf4e3125]{
 -- Page 391, Offset 36417.
 -- 白い吸血鬼に集中する火線。
-A line of fire focused on the vampire in white.
+A firing line centered on the vampire in white.
 }
 [sha:54960deb15dce3bda0247fd0dac5aa6236846aa4]{
 -- Page 391, Offset 36418.
 -- 少年の知る彼女なら確実に殺しきれるソレは、赤い瞳の微笑みだけで霧散した。
-If she had still been the woman he once knew, she would've certainly died right then and there. However, her red eyes and smile alone were enough to make that possibility vanish completely.
+// Not entirely clear on what exactly ソレ is referring to here, as well as the target for 霧散する
+// I guess it's the 集中する火線? Kinda vague, so TLing it as his plan
+If she'd still been the woman he once knew, she really would have died right then and there.
+ Instead, her red eyes and smile are all it takes to unravel his plan.
```

Comment:
I've talked with about 3 friends about this, and everyone says that reading is cannot be correct. At the very least, it's closer to whatever the truth is.

Reply:
Okay talking to more friends, it gets more confusing. Everyone has different interpretations

Reply:
Yeah confusing for sure. It feels weird for it to be something she does to herself though because of the last bit - why would her eyes be doing something to _her_?

Reply:
Well, after a bit of investigation, I think the original interpretation is more accurate.
36393: 少年の用意した戦力は吸血鬼アルクェイドを打倒できるものだった。上回ってはいないが、**殺し切る**に足る道筋が用意されている。

So rather than this line of him knowing her being implied to be from the past, it's more about what he'd expected about her in this specific encounter. So yes, I guess the ソレ is the 火線,

赤い瞳の微笑みだけで is probably not too literal, and why that part is a bit confusing. It's probably just something that occurs concurrently, but for prose purposes he wanted it to sound like this. 霧散する then kinda has to be the ソレ 'dispersing' or whatever. Probably because of the "invisible flower", which I think is some kind of weird description of her Marble Phantasm

Reply:
>Had she been the woman Mario thought her to be, that would've proven enough to kill her, 
>but instead, her red eyes and smile are all it takes for it to be reduced to nothing.

I think put like this, it doesn't need to be so oblique with それ, because it's much clearer in this EN sentence what the 'that' is. The second sentence is a bit stappling still, because 'her red eyes and smile are all it takes' is just a bit weird, but hey, why not?